src/main/java/com/prgrms/needit/domain/member/dto/MemberCreateRequest.java

```diff
@@ -0,0 +1,34 @@
+package com.prgrms.needit.domain.member.dto;
+
+import com.prgrms.needit.domain.member.entity.Member;
+import javax.validation.constraints.NotBlank;
+import lombok.Getter;
+
+@Getter
+public class MemberCreateRequest {
+
+	@NotBlank
+	private String email;
+	@NotBlank
+	private String nickname;
+	@NotBlank
+	private String password;
+	@NotBlank
+	private String address;
+	@NotBlank
+	private String contact;
+	@NotBlank
+	private String profileImageUrl;
+
+	public Member toEntity() {
+		return Member.builder()
+					 .email(this.email)
+					 .nickname(this.nickname)
+					 .password(this.password)
```

Comment:
비밀번호 암호화해주세용 [참고 블로그](https://youngjinmo.github.io/2021/05/passwordencoder/)
```suggestion
					 .email(this.email)
					 .nickname(this.nickname)
					 .password(passwordEncoder.encode(this.password))
```